src/SystemCommands-MessageCommands/SycReplaceMessageCallWithOtherCommand.class.st

```diff
@@ -0,0 +1,53 @@
+Class {
+	#name : #SycReplaceMessageCallWithOtherCommand,
```

Comment:
And here it would be SycReplaceMessageCommand

Reply:
Thanks, I already changed the names